bblfsh/test.py

```diff
@@ -8,6 +8,48 @@
 from bblfsh.launcher import ensure_bblfsh_is_running
 
 
+from bblfsh import Node as NodeNative
+
+class XpathTests(unittest.TestCase):
+    def testXpath(self):
+        NodeNative.initialize()
+
+        root = NodeNative("compilation_unit")
+        node1 = NodeNative("class")
+        node2 = NodeNative("identifier")
+        node2.token = "first"
+        node2.add_role(1)
+        node2.add_role(2)
+        node2.add_role(3)
+
+        node3 = NodeNative("block")
+        node4 = NodeNative("method")
+        node5 = NodeNative("identifier")
+        node5.token = "second"
+        node6 = NodeNative("block")
+        node8 = NodeNative("loop")
+
+        root.add_child(node1)
+
```

Comment:
Spurious whitespace?, probably all the add_child statements would be better on a single logica paragraph (without whitespace between them).